requirements/default.txt

```diff
@@ -166,6 +181,9 @@ ordereddict==1.1
 # sha256: Rsjbda51oFa9HMB_ohc0_i5gPRGgeDPswe63TDXHLgw
 pbr==1.8.1
 
+# sha256: DxedfnXnyDtjQblZXKHzlN5wgUhKnjUq1m1VOhw9qik
+Pillow==2.8.2
```

Comment:
According to peep, this has the wrong hash:

```
THE FOLLOWING PACKAGES DIDN'T MATCH THE HASHES SPECIFIED IN THE REQUIREMENTS
FILE. If you have updated the package versions, update the hashes. If not,
freak out, because someone has tampered with the packages.

    Pillow: expected DxedfnXnyDtjQblZXKHzlN5wgUhKnjUq1m1VOhw9qik
                 got LK9bmSvb5n4FCEZYt4PU3LnPe-zaN5TZXIngV1xFPDo

-------------------------------
Not proceeding to installation.
```

I think the version got downgraded (2.9 -> 2.8.2), but the hash didn't get updated.